src/formatDate.js

```diff
@@ -50,7 +50,33 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const resultArr = Array(3);
+  const oldSeparator = fromFormat[3];
+  const newSeparator = toFormat[3];
+  const dateArr = date.split(oldSeparator);
+  const dayPosition = fromFormat.findIndex(e => e.startsWith('D'));
+  const monthPosition = fromFormat.findIndex(e => e.startsWith('M'));
+  const yearPosition = fromFormat.findIndex(e => e.startsWith('Y'));
+  const dayNewPosition = toFormat.findIndex(e => e.startsWith('D'));
+  const monthNewPosition = toFormat.findIndex(e => e.startsWith('M'));
+  const yearNewPosition = toFormat.findIndex(e => e.startsWith('Y'));
```

Comment:
You could try to rewrite your code.

Create empty objectData for example and then add there this values using `for` loop.

This way would be kind of shorter.

Reply:
Thanks) This is really a good idea))